### PR TITLE
Validate context attribute values

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag @jframos 

### DESCRIPTION
- Adding exception control to some no-managed situations.
- Adding tests to check the use case: exception control to some no-managed situations.

### CODE REVIEW

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
Is this PR fixing the issue *CLAUDIA-5524* ?

**Comment by geonexus** [comment]
yes

File: tests/test_facts.py [hunk 1]
```diff
@@ -145,3 +145,111 @@ def test_context_broker_message(self):
 
         response = self.app.post(self.url3, data=json.dumps(data2), headers={'Content-Type': 'application/json'})
         self.assertEqual(response.status_code, 200)
+
+    def test_context_broker_empty_attributes_message(self):
+        """ Test that the POST operation over the API returns a 400 error response if message has no attributes.
+
+        :return       200 Ok
```
Comment (jframos) [inline tests/test_facts.py]: remove this line

File: tests/test_facts.py [hunk 2]
```diff
@@ -145,3 +145,111 @@ def test_context_broker_message(self):
...
+                ]
+            }
+
+        response = self.app.post(self.url3, data=json.dumps(data3), headers={'Content-Type': 'application/json'})
+        self.assertEqual(response.status_code, 400)
+
+    def test_context_broker_any_attribute_is_missing_message(self):
+        """ Test that the POST operation over the API returns a 400 error response if message has no attributes.
+
+        :return       200 Ok
```
Comment (jframos) [inline tests/test_facts.py]: remove this line

File: tests/test_facts.py [hunk 3]
```diff
@@ -145,3 +145,111 @@ def test_context_broker_message(self):
...
+                ]
+            }
+
+        response = self.app.post(self.url3, data=json.dumps(data4), headers={'Content-Type': 'application/json'})
+        self.assertEqual(response.status_code, 400)
+
+    def test_context_broker_message_attribute_out_of_range_value(self):
+        """ Test that the POST operation over the API returns a valid response if message is built correctly.
+
+        :return       200 Ok
```
Comment (jframos) [inline tests/test_facts.py]: remove this line

File: tests/test_facts.py [hunk 4]
```diff
@@ -145,3 +145,111 @@ def test_context_broker_message(self):
...
+                       }
+                   }
+                ]
+            }
+
+        response = self.app.post(self.url3, data=json.dumps(data4), headers={'Content-Type': 'application/json'})
+        self.assertEqual(response.status_code, 400)
+
+    def test_context_broker_message_attribute_out_of_range_value(self):
+        """ Test that the POST operation over the API returns a valid response if message is built correctly.
```
Comment (jframos) [inline tests/test_facts.py]: C&P?

**Comment by jframos** [comment]
I think we could add a new unit test for one of the detected issue:
- Type validation (value should be a 'float')

Acceptance tests (negative test cases) should be updated taking into account the new valid range (0.0000 to 100.000')

**Comment by jframos** [comment]
LGTM

**Pull request merged**